src/yuzu/main.cpp

```diff
@@ -2840,170 +2840,335 @@ void GMainWindow::OnGameListNavigateToGamedbEntry(u64 program_id,
+        } else {
+            LOG_ERROR(Frontend, "Failed to create shortcut");
+        }
+        shortcut_stream.close();
+    } catch (const std::exception& e) {
+        LOG_ERROR(Frontend, "Failed to create shortcut: {}", e.what());
     }
-    case GameListShortcutTarget::Applications: {
-        const QString applications_path =
-            QStandardPaths::writableLocation(QStandardPaths::ApplicationsLocation);
-        if (applications_path.isEmpty()) {
-            const char* home = std::getenv("HOME");
-            if (home != nullptr) {
-                target_directory = std::filesystem::path(home) / ".local/share/applications";
+    return false;
+#elif defined(_WIN32) // Windows
+    HRESULT hr = CoInitialize(NULL);
+    if (SUCCEEDED(hr)) {
+
+        IShellLinkW* ps1 = nullptr;
+        IPersistFile* persist_file = nullptr;
+
+        SCOPE_EXIT({
+            if (persist_file != nullptr) {
+                persist_file->Release();
+            }
+            if (ps1 != nullptr) {
+                ps1->Release();
+            }
+
+            CoUninitialize();
+        });
+
+        HRESULT hres = CoCreateInstance(CLSID_ShellLink, NULL, CLSCTX_INPROC_SERVER,
+                                        IID_IShellLinkW, (void**)&ps1);
+
+        std::wstring wshortcut_path_full =
+            Common::UTF8ToUTF16W((shortcut_path_full).string() + ".lnk");
+        std::wstring wicon_path = Common::UTF8ToUTF16W(icon_path.string());
+        std::wstring wcommand = Common::UTF8ToUTF16W(command.string());
+        std::wstring warguments = Common::UTF8ToUTF16W(arguments);
+        std::wstring wcomment = Common::UTF8ToUTF16W(comment);
+
+        if (SUCCEEDED(hres)) {
+            hres = ps1->SetPath(wcommand.data());
+
+            if (SUCCEEDED(hres) && !arguments.empty()) {
+                hres = ps1->SetArguments(warguments.data());
+            }
+
+            if (SUCCEEDED(hres) && !comment.empty()) {
+                hres = ps1->SetDescription(wcomment.data());
+            }
+
+            if (SUCCEEDED(hres) && std::filesystem::is_regular_file(icon_path)) {
+                hres = ps1->SetIconLocation(wicon_path.data(), 0);
+            }
+
+            if (SUCCEEDED(hres)) {
+                hres = ps1->QueryInterface(IID_IPersistFile, (void**)&persist_file);
+            }
+
+            if (SUCCEEDED(hres) && persist_file != nullptr) {
+                hres = persist_file->Save(wshortcut_path_full.data(), TRUE);
+                if (SUCCEEDED(hres)) {
+                    return true;
+                } else {
+                    LOG_ERROR(Frontend, "Failed to create shortcut");
+                }
             }
         } else {
-            target_directory = applications_path.toUtf8().toStdString();
+            LOG_ERROR(Frontend, "Failed to create CoCreateInstance");
         }
-        break;
     }
+    return false;
+#else                 // Unsupported platform
+    return false;
+#endif
+}
```

Comment:
Function catch block:
```suggestion
} catch (std::exception& e) {
    LOG_ERROR(Frontend, "Failed to create shortcut: {}", e.what());
    return false;
}
```